src/main/java/baritone/pathing/movement/Moves.java

```diff
@@ -322,6 +322,54 @@ public Movement apply0(CalculationContext context, BetterBlockPos src) {
         public void apply(CalculationContext context, int x, int y, int z, MutableMoveResult result) {
             MovementParkour.cost(context, x, y, z, EnumFacing.WEST, result);
         }
+    },
+
+    PARKOUR_ADV_NORTH(0, 0, 0, true, true) {
```

Comment:
BTW can you change this to instead be the max distance along north? Because of `if ((newX >> 4 != currentNode.x >> 4 || newZ >> 4 != currentNode.z >> 4) && !calcContext.isLoaded(newX, newZ)) {` in AStarPathFinder

It **probably** is okay, becuase parkour will fail if the destination is just air, but it will probably help prevent potential weird behavior. Maybe has not come up because I imagine you aren't testing this with long-distance pathing that much, and mostly just short range?